c2corg_ui/static/js/search/pagination.js

```diff
@@ -47,6 +47,12 @@ app.PaginationController = function($scope, ngeoLocation) {
 
   /**
    * @type {number}
+   * @private
+   */
+  this.maxResultOffset_ = 10000;
```

Comment:
I don't have a strong opinion about this but since this is a constant, it could be a static property with a ``@const`` keyword as for instance in https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/map/map.js#L348-L352